ZenPacks/zenoss/Import4/service_definition/imp4opentsdb.json

```diff
@@ -0,0 +1,107 @@
+{
+   "servicePath": "/",
+   "serviceDefinition": {
+        "Name": "Imp4OpenTSDB",
+        "Launch": "manual",
+        "Command": "bash -c \"while echo 'Starting ..'; do [ -f /import4/imp4opentsdb.sh ] && /import4/imp4opentsdb.sh || echo 'Startup script not ready ...'; sleep 7; done;\"",
+        "Description": "OpenTSDB environment for import4",
+        "Services": [],
+        "Tags": [
+            "daemon"
+        ],
+        "ImageID": "zenoss/opentsdb:v11",
```

Comment:
This is going to change over time.  How do we want to handle keeping up with the core changes?